sdk/utils.ts

```diff
@@ -188,6 +193,66 @@ export const fetchJwtToken = async (claims: {}, expiresAt?: Date): Promise<{toke
   }
 }
 
-export const decodeJwtToken = (token: string) => {
-  return JSON.parse(Buffer.from((token as string).split('.')[1], 'base64').toString())
+interface DecodedJwt {
+  header: any,
+  payload: any
+}
+
+export const decodeJwt = (token: string): DecodedJwt => {
+
+  const splitJwt = (token).split('.');
+  const header = JSON.parse(Buffer.from(splitJwt[0], 'base64').toString())
+  const payload = JSON.parse(Buffer.from(splitJwt[1], 'base64').toString())
+
+  return {
+    header,
+    payload
+  }
+}
+
+const DISPTACHER_ARE_THEY_LINKED = "function areTheyLinked(uint256 agentId, uint256 scannerId) external view returns(bool)";
+const DISPATCH_CONTRACT = "0xd46832F3f8EA8bDEFe5316696c0364F01b31a573"; // Source: https://docs.forta.network/en/latest/smart-contracts/
+
+export const verifyJwt = async (token: string): Promise<boolean> => {
+  const splitJwt = (token).split('.')
+  const rawHeader = splitJwt[0]
+  const rawPayload = splitJwt[1]
+
+  const header = JSON.parse(Buffer.from(rawHeader, 'base64').toString())
+  const payload = JSON.parse(Buffer.from(rawPayload, 'base64').toString())
+
+  const botId = payload["bot-id"] as string
+  const algorithm = header?.alg;
+
+  if(algorithm !== "ETH") {
+    console.warn(`Unexpected signing method: ${algorithm}`)
+    return false
+  }
+
+  if(!botId) {
+    console.warn(`Invalid claim`)
+    return false
+  }
+
```

Comment:
we also need to verify token expiry timestamp somewhere here

Reply:
Added a check for the expiry time